csp-guru/README.md

```diff
@@ -0,0 +1,50 @@
+# CSP.guru
+A database of concentrating solar power plants of the world for energy modellers and analysts.
+## About
+This dataset holds technical, economic, financial, and industrial data on all operating concentrating solar power stations and of stations under construction (larger than 10 MW). For detailed description of the data collection procedure, please see the original article [Lilliestam et al. (2017)](https://doi.org/10.1038/nenergy.2017.94).
+## Data Package
+We provide the data in form of a [tabular data package](https://frictionlessdata.io/specs/tabular-data-package/) that consists of a CSV (comma separated values) file holding the actual data and a single JSON file (`datapackage.json`) holding the metadata.
+## License and conditions
+The licence of this data allows academic and all non-commercial use if you give appropriate credit, under the [Creative Commons Attribution-NonCommercial 4.0 International license](https://creativecommons.org/licenses/by-nc/4.0/).
+For academic and professional use (presentations, journal articles, reports, etc):
+Please cite the original source, the paper [Lilliestam et al. (2017)](https://doi.org/10.1038/nenergy.2017.94). Please also link to www.csp.guru if possible.
+## Changes in this version of CSP.guru
+The data collection for this version of CSP guru has been conducted by Johan Lilliestam and Richard Thonig in the first half of 2019. It is based on the [May 2018 Version]( https://doi.org/10.5281/zenodo.1342716 ) by Johan Lilliestam, Mercè Labordena and Lana Ollier. The data reflects the situation of CSP projects and industry on January first of 2019. For this iteration of the dataset we decided to limit it to materializing projects, i.e. those under construction and those already commissioned. Moreover, we added the status of decommissioned plant, as both SEGS I and II have reached the end of their lifetime. Additionally, the following changes in the structure of the data were implemented compared to the last version.
+
+#### New Columns
```

Comment:
tiny thing, but why level 4 headline instead of level 3?

Reply:
changed it. I felt it looked better with level4